react/features/chat/components/native/Chat.js

```diff
@@ -0,0 +1,196 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Modal, SafeAreaView } from 'react-native';
+import { GiftedChat } from 'react-native-gifted-chat';
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+
+import { translate } from '../../../base/i18n';
+import { getLocalParticipant } from '../../../base/participants';
+import { BackButton, Header, HeaderLabel } from '../../../base/react';
+
+import { sendMessage, toggleChat } from '../../actions';
+
+import styles from './styles';
+
+/**
+ * The type of the React {@code Component} props of {@link Chat}.
+ *
+ * Co-authored with https://github.com/dgeorgiev06
```

Comment:
This comment here makes no sense. This comment describes the Props. Please put it as `Co-authored-by` in your commit message. Here is an example: ea22d125818920037306b5121eb193ab6246a95a